dask_kubernetes/operator/customresources/daskworkergroup.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: daskworkergroups.kubernetes.dask.org
+spec:
+  scope: Namespaced
+  group: kubernetes.dask.org
+  names:
+    kind: DaskWorkerGroup
+    plural: daskworkergroups
+    singular: daskworkergroup
+    shortNames:
+      - daskworkers
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
+            status:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
```

So now I need to fill out this spec section more. @jacobtomlinson 
```suggestion
            spec:
              type: object
              properties:
                replicas:
                  type: integer
                image:
                  type: string  
              x-kubernetes-preserve-unknown-fields: true
            status:
              type: object
              x-kubernetes-preserve-unknown-fields: true
```

---

Sure. I'm tempted to suggest we leave things with the `x-kubernetes-preserve-unknown-fields: true` for now and come back and fully nail down the spec later.

Once things are finished we shouldn't have `x-kubernetes-preserve-unknown-fields: true` anywhere (and should probably add a test for that at some point).